samples/Mvc.RenderViewToString/Views/EmailPartial.cshtml

```diff
@@ -0,0 +1,4 @@
+﻿@inject Mvc.RenderViewToString.EmailReportHelper ReportHelper
+
+    This is the partial view content.<br />
+    @ReportHelper.GenerateReport(Model.UserData1, Model.UserData2).<br />
```

Comment:
It would. But we already use partial views here and adding a view component as well feels unnecessary.